rules/cross-platform/multiple_alerts_involving_user.toml

```diff
@@ -17,21 +17,29 @@ false_positives = [
     activity, or an upcoming incident.
     """,
 ]
-from = "now-24h"
-index = [".alerts-security.*"]
+from = "now-4h"
```

Comment:
Remember no deduplication for ESQL if aggregation based upstream. We can either recommend the customer add suppression for this rule if they'd like or keep a shorter interval window.